code/style.css

```diff
@@ -1,4 +1,380 @@
+}
+
+.big-news-text {
+  padding-top: 20px;
+}
+
+.big-news-text-container {
+  padding: 0px 10px 20px 10px;
+}
+
+/* ---------- OTHER NEWS CONTAINER ------------- */
+
+.other-news-container {
+  display: flex;
+  flex-direction: column;
+  max-width: 100%;
+}
+
+.other-news {
+  background-color: rgb(240, 236, 236);
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  margin: 0px 0px 20px 0px;
+}
+
+.other-news-pictures {
+  height: 130px;
+  max-width: 50%;
+  flex: 1;
+  padding: 0px 10px 0px 10px;
+  object-fit: cover;
+}
+
+.other-news-text-container {
+  display: flex;
+  flex-direction: column;
+  padding: 0px 10px 0px 10px;
+  flex: 1;
+  height: 130px;
+}
+
+.other-news-bold {
+  font-weight: 700;
+  font-size: 13px;
+}
+
+.other-news-thin {
+  font-weight: 400;
+  font-size: 13px;
+}
+
+/* ---------- DAILY-POLAROIDS ------------- */
+
+.daily-polaroids {
+  background-color: rgb(240, 236, 236);
+}
+
+.daily-polaroid-container {
+  display: flex;
+  justify-content: center;
+  flex-flow: row wrap;
+}
+
+.daily-polaroid-header {
+  text-align: center;
+  padding-top: 20px;
+  padding-bottom: 40px;
+}
+
+.daily-polaroid {
+  height: 300px;
+  width: 250px;
+  object-fit: cover;
+  display: block;
+}
+
+.polaroid {
+  background: #ffffff;
+  display: inline-block;
+  margin-bottom: 20px;
+  padding: 15px 15px 30px;
+  text-align: center;
+  text-decoration: none;
+  -webkit-box-shadow: 0 4px 6px rgba(0, 0, 0, 0.3);
+  -moz-box-shadow: 0 4px 6px rgba(0, 0, 0, 0.3);
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.3);
+  -webkit-transition: all 0.2s linear;
+  -moz-transition: all 0.2s linear;
+  transition: all 0.2s linear;
+}
+
+.polaroid-text {
+  padding-top: 20px;
+}
+
+.flip-card-back {
+  display: none;
+}
+
+/* ---------- IPAD ADAPTATIONS------------- */
+
+@media only screen and (min-width: 668px) and (max-width: 1024px) {
+  .header {
+    justify-content: space-evenly;
+  }
+
+  .header :nth-child(1) {
+    order: 2;
+  }
+  .header :nth-child(2) {
+    order: 0;
+  }
+  .header :nth-child(3) {
+    order: 1;
+  }
+  .header :nth-child(4) {
+    order: 3;
+  }
+  .header :nth-child(5) {
+    order: 4;
+  }
+
+  .icon-and-text {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    margin-right: 20px;
+  }
+
+  .big-news-container {
+    flex-direction: row;
+    margin-bottom: 30px;
+  }
+
+  .big-news-image {
+    flex: 1;
+    height: 385px;
+    object-fit: cover;
+  }
+
+  .big-news-text-container {
+    flex: 1;
+    padding: 0px 20px 0px 20px;
+  }
+
+  .other-news-container {
+    flex-flow: row wrap;
+    justify-content: space-evenly;
+  }
+
+  .other-news {
+    flex-direction: column;
+    width: auto;
+    max-width: none;
+  }
+
+  .other-news-pictures {
+    max-width: 100%;
+    max-height: 200px;
+    margin-top: 10px;
+  }
+  .other-news-text-container {
+    max-width: 300px;
+    margin-top: 10px;
+    height: auto;
+  }
+
+  .item-2,
+  .item-4,
+  .item-6 {
```

Comment:
They seem to have the same styling. Maybe could have given them the same name, just to minimize the number of classes?

Reply:
Very true! =) changed this!